src/cpp/src/gguf_utils/gguf_modeling.cpp

```diff
@@ -152,24 +152,43 @@ std::shared_ptr<ov::Model> create_language_model(
 
 } // namespace
 
-std::shared_ptr<ov::Model> create_from_gguf(const std::string& model_path) {
+void save_openvino_model(const std::shared_ptr<ov::Model>& model, const std::string& save_path, bool compress_to_fp16) {
+    try {
+        auto serialize_start_time = std::chrono::high_resolution_clock::now();
+        ov::save_model(model, save_path, compress_to_fp16);
+        auto serialize_finish_time = std::chrono::high_resolution_clock::now();
+        auto serialize_duration = std::chrono::duration_cast<std::chrono::milliseconds>(serialize_finish_time - serialize_start_time).count();
+        std::cout << "Save generated OpenVINO model to: " << save_path << " done. Time: " << serialize_duration << " ms\n";
```

Comment:
I agree that `save_path` print is helpful, but it still would be strange to see this log on every GGU pipeline construction. That's the documentation is for. Add an entry to https://github.com/openvinotoolkit/openvino.genai/blob/dad32c5adb8ac396d12a05831c34f2f5e77d8694/src/cpp/include/openvino/genai/llm_pipeline.hpp#L329
and remove this print. I think `extern const bool ENABLE_SAVE_OV_MODEL;` from utils.hpp can be removed after that.